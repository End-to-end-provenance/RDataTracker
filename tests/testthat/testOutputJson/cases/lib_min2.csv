"package","version"
"base","3.5.1"
"RDataTracker","2.27.0"
