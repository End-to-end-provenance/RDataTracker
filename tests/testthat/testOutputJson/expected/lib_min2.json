
		// library nodes - prov collections
		"rdt:l1": {
			"name": "base",
			"version": "3.5.1",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		},
		"rdt:l2": {
			"name": "RDataTracker",
			"version": "2.27.0",
			"prov:type": {
				"$": "prov:Collection",
				"type": "xsd:QName"
			}
		}
